### PR TITLE
[BFCL] Add New Model `watt-tool-8B` and `watt-tool-70B`

### PR DESCRIPTION
**Pull request by zhanghanduo**
This PR adds the model `watt-ai/watt-tool-8B` and `watt-ai/watt-tool-70B` to the leaderboard.

### CODE REVIEW

**Review by HuanzhiMao — APPROVED** [review]
Thanks for the PR @zhanghanduo!
Once you sync this branch with the latest changes from the main branch, I’ll be happy to merge it. Currently there are some conflicts that are blocking the merge, and I don't have write access to your branch.

**Comment by zhanghanduo** [comment]
> Thanks for the PR @zhanghanduo! Once you sync this branch with the latest changes from the main branch, I’ll be happy to merge it. Currently there are some conflicts that are blocking the merge, and I don't have write access to your branch.

Hi, thanks for pointing out that conflict. I have merged the update that happened after our submission.

**Pull request merged**
